src/libNode/MicroBlockPreProcessing.cpp

```diff
@@ -641,9 +646,22 @@ void Node::ProcessTransactionWhenShardBackup() {
 
   cv_TxnProcFinished.notify_all();
 
+  PutTxnsInTempDataBase(t_processedTransactions);
```

Comment:
We need to consider the fact that the txns in leveldb for backup may not be confirmed transaction.